Java/tests/src/test/java/com/microsoft/recognizers/text/tests/numberwithunit/NumberWithUnitTest.java

```diff
@@ -35,7 +35,7 @@ protected void recognizeAndAssert(TestCase currentCase) {
         List<ModelResult> results = recognize(currentCase);
 
         // assert
-        assertResultsWithKeys(currentCase, results, getKeysToTest(currentCase));
+        assertResults(currentCase, results, getKeysToTest(currentCase));
```

Comment:
Just to know, NumberWithUnit doesn't validate for `start` and `end`? Or maybe this was not currently applied?

Reply:
Great, so this will consume the assertModel of the `AbstractTest`, right?